### PR TITLE
[monitor-gcp] Fix Service Account Roles

### PR DESCRIPTION
**Pull request by andremm**
The _Monitor Google Cloud Platform_ tutorial is missing one role in the GCP service account setup. Without the **Pub/Sub Subscriber** role Filebeat cannot collect audit logs. This PR fixes the tutorial to also include this role when setting up the service account.

### CODE REVIEW

File: docs/en/observability/monitor-gcp.asciidoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ image::monitor-gcp-service-account-name.png[Service account name]
 Make sure to select the correct roles.
 ====
 
-To monitor {gcp} services, you need to add two main roles to the
+To monitor {gcp} services, you need to add three main roles to the
```
Comment (dedemorton) [inline docs/en/observability/monitor-gcp.asciidoc:56]:
```suggestion
To monitor {gcp} services, you need to add these roles to the
```

Comment (dedemorton) [reply docs/en/observability/monitor-gcp.asciidoc:56]: ...this way you won't have to update this again if the number changes. :-)

**Review by dedemorton — APPROVED** [review]
One minor change (optional, but recommended). Otherwise looks good to me, so I will mark as approved.

**Comment by andremm** [comment]
@dedemorton thanks for your suggestion. I just applied it. Should I merge this PR?

**Comment by dedemorton** [comment]
@andremm Just wait until the build passes. Let me know if you want me to backport this and which versions you want it in.

**Comment by andremm** [comment]
thanks @dedemorton. I will try to backport this. I let you know if I need help. :)

**Pull request merged**